snappi_ixnetwork/timer.py

```diff
@@ -1,5 +1,7 @@
 import time
 
+timer_data = {}
+
 
 class Timer:
```

Comment:
as a lite note please do not use single char as variable name when that will use in multiple line.

Reply:
I did not get his Alakendu, can you give some example